README.md

```diff
@@ -14,4 +14,11 @@ MCR.Developer.Resource.Override=/path/to/reposis_artus/src/main/resources
 MCR.LayoutService.LastModifiedCheckPeriod=0
 MCR.UseXSLTemplateCache=false
 MCR.SASS.DeveloperMode=true
+```
+
+For SOLR
+in [$MIR-HOME]/data/solr/configsets/mycore_main/conf/managed-schema add the following field:
+
+```
+<field name="artus.sections" type="string" indexed="true" stored="true" multiValued="false"/>
```

Comment:
do not change managed-schema!
use mycore cli with: `reload solr configuration main in core main`